README.rst

```diff
@@ -188,8 +188,9 @@ the **BSD 3-clause License**. A copy of this license is provided in
 Support
 -------
 
-The development of PyGMT was funded by
-`NSF grant OCE-1558403 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1558403>`__.
+The development of PyGMT was supported by NSF grants
```

Comment:
Should we change `was` to `was/is`?

Reply:
Yeah I thought about that. Probably should.

Reply:
@PaulWessel sorry to bother you with this tiny linguistic issue.

Reply:
I think you're looking for "has been" :)

Reply:
Present perfect, thanks! Don't tell my English teacher.